Tests/RxCocoaTests/DelegateProxyTest.swift

```diff
@@ -446,6 +447,9 @@ extension DelegateProxyTest {
         autoreleasepool {
             control = nil
         }
+        
+        wait(for: [expectation], timeout: 1)
```

Comment:
why is this needed? seems like the tests pass without it as well

Reply:
WKWebView is not deallocated immediately. Internally there is something else that retains the WKWebView for a little longer than expected:

https://stackoverflow.com/questions/39349911/memory-leak-in-wkwebview-instantiation